src/Components/WeatherCard/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC } from "react";
+import { useHistory } from "react-router-dom";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import Avatar from "@mui/material/Avatar";
+import { dateToLocalDay } from "../../constant";
+import { CityInterface } from "./weatherCard.interface";
+import { useStyles } from "./style";
+
+const WeatherCard: FC<{ data: any; city: CityInterface }> = ({
```

Comment:
Don't use any and define proper type for it

Reply:
Remove as much as possible